eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Serialisation/CDDLUtils.hs

```diff
@@ -189,3 +191,6 @@ verifyConforming value cddl =
     >>= \case
       Right _ -> pure True
       Left _ -> pure False
+
+throwIO :: (Trans.MonadIO m, Exception e) => e -> m a
+throwIO = liftIO . PException.throwIO
```

Comment:
We already depend on a library `unliftio` that provides such lifted function: [`throwIO`](https://www.stackage.org/haddock/nightly-2022-05-20/unliftio-0.2.22.0/UnliftIO-Exception.html#v:throwIO), so there is no need to redefine, we can just import it.